src/vs/workbench/services/extensions/common/extensions.ts

```diff
@@ -359,10 +359,11 @@ function extensionDescriptionArrayToMap(extensions: IExtensionDescription[]): Ma
 }
 
 export function isProposedApiEnabled(extension: IExtensionDescription, proposal: ApiProposalName): boolean {
-	if (!extension.enabledApiProposals) {
-		return false;
-	}
-	return extension.enabledApiProposals.includes(proposal);
+	return true;
+	// if (!extension.enabledApiProposals) {
+	// 	return false;
+	// }
+	// return extension.enabledApiProposals.includes(proposal);
```

Comment:
Not needed proposed API are already enabled https://github.com/gitpod-io/openvscode-server/commit/750111b78a2360845585cb0b737bc9eda803109d